nomenclature/cli.py

```diff
@@ -277,3 +277,30 @@ def cli_run_workflow(
     df = getattr(workflow, workflow_function)(IamDataFrame(input_file))
     if output_file is not None:
         df.to_excel(output_file)
+
+
+@cli.command("validate-scenarios")
+@click.argument("input_file", type=click.Path(exists=True, path_type=Path))
+@click.option(
+    "--definitions",
+    help="Optional name for definitions folder",
+    type=click.Path(exists=True, path_type=Path),
+    default="definitions",
+)
+def cli_validate_scenarios(input_file: Path, definitions: Path):
+    """Run a given input file through a DataStructureDefinition validation,
+    checking it against defined codelist(s)
```

Comment:
Try to keep the docstring such that the first line is really only one line, per numpydoc standards.
```suggestion
    """Validate a scenario file against the codelists of a project
```